src/test/java/com/stripe/StripeMockProcess.java

```diff
@@ -50,7 +50,8 @@ public static boolean start() throws IOException, InterruptedException, NoSuchFi
             "-spec",
             getPathSpec(),
             "-fixtures",
-            getPathFixture()
+            getPathFixture(),
+            "-strict-version-check"
```

Comment:
I assume we want to pass strict version check when we do custom build too? Or do we want to me more lenient here as we are developing and might be testing against the latest OpenAPI spec?